apps/full-site-editing/full-site-editing-plugin/editor-domain-picker/src/domain-picker-button/index.tsx

```diff
@@ -22,16 +24,24 @@ declare global {
 export default function DomainPickerButton() {
 	const [ isDomainModalVisible, setDomainModalVisibility ] = React.useState( false );
 	const [ domainSearch, setDomainSearch ] = React.useState( '' );
-
 	const site = useSelect( ( select ) => select( SITES_STORE ).getSite( window._currentSiteId ) );
+	const [ domainName, setDomainName ] = React.useState(
+		site?.URL && new URL( site?.URL ).hostname
```

Comment:
Note that IE11 is not supported for the `URL API`. https://caniuse.com/#feat=url
This is temporary anyway for now until we figure out how to extract site name.